src/DependencyInjection/LeezyPheanstalkExtension.php

```diff
@@ -22,10 +25,11 @@ class LeezyPheanstalkExtension extends Extension
     public function load(array $configs, ContainerBuilder $container)
     {
         $configuration = new Configuration();
-        $config = $this->processConfiguration($configuration, $configs);
+        $config        = $this->processConfiguration($configuration, $configs);
 
         $loader = new Loader\XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
         $loader->load('services.xml');
+        $loader->load('commands.xml');
 
         if (!$config['enabled']) {
```

Comment:
This condition breaks on ```leezy_pheanstalk.enabled: false``` because ```LeezyPheanstalkExtension::configureConnections()``` isn't called, then, definition for ```leezy.pheanstalk.pheanstalk_locator``` is never created.

The results:
```
$ app/console cache:clear -e prod
Clearing the cache for the prod environment with debug false


                                                                                                                                       
  [Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException]                                                           
  The service "leezy.pheanstalk.command.delete_job" has a dependency on a non-existent service "leezy.pheanstalk.pheanstalk_locator".  
                                                                                                                                       
```

ping @pkruithof 
/cc @armetiz